.github/CODEOWNERS

```diff
@@ -14,4 +14,4 @@
 /quickstarts/whereami/ @theemadnes @yoshi-approver @GoogleCloudPlatform/dee-platform-ops
 /security/language-vulns/ @yoshi-approver @GoogleCloudPlatform/dee-platform-ops
 /streaming/ @aburhan @pwschuurman @yoshi-approver @GoogleCloudPlatform/dee-platform-ops
-/windows/ @ibabou @mcshooter @yoshi-approver @GoogleCloudPlatform/dee-platform-ops
+/windows/ @ibabou @yoshi-approver @GoogleCloudPlatform/dee-platform-ops
```

Comment:
I've removed mcshooter because this GitHub account no longer in the GoogleCloudPlatform org.
![Screenshot 2024-10-08 at 3 20 52 PM](https://github.com/user-attachments/assets/825bc438-6b28-4b8e-9d44-928451223ea4)